homeassistant/components/intellidrive/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
+        },
+        "error": {
+            "cannot_connect": "Failed to connect",
+            "invalid_auth": "Invalid authentication",
+            "unknown": "Unexpected error"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "device_key": "Device key",
+                    "host": "Host",
+                    "password": "Password",
+                    "username": "Username"
+                }
+            }
+        }
+    }
+}
```

Comment:
This file should no longer be pushed.

Reply:
so should I delete it?

Reply:
I have one more question:
Although I use the predefined constants of HA (as well as CONF_HOST) only the text field for the host is filled with the corresponding title in the setup, the token text field is without title. What can be the reason for this?

Reply:
You need to regenerate the translation files:
```
python3 -m script.translations develop --all
```

Also, you need to force-refresh the frontend to make it pick up the changed translations. The easiest way is to use your browser's developer mode with cache disabled.

Reply:
Should I delete the translations/en.json file?